libtado/api.py

```diff
@@ -54,20 +56,27 @@ def __init__(self, username, password, secret):
 
   def _login(self):
     """Login and setup the HTTP session."""
+    print(pkg_resources.require)
+    print(metadata.version)
+    print(metadata.version('libtado'))
     url='https://auth.tado.com/oauth/token'
+    headers = { 'User-Agent': 'python/libtado' }
```

Comment:
The version number is not included despite that being suggested in the PR title.

Reply:
Identifying the library is fait enough for Tado. As they don't manage this library, they don't need to know the lib version.